oggsound/src/OgreOggSoundManager.cpp

```diff
@@ -215,7 +215,10 @@ namespace OgreOggSound
 		if ( mListener )
 		{
 			Ogre::SceneManager* s = mListener->getSceneManager();
-			s->destroyAllMovableObjectsByType("OgreOggISound");
+
+			if(s != nullptr && Ogre::Root::getSingletonPtr()->hasSceneManager(s->getName()))
```

Comment:
this will crash in getName, as we will have a dangling pointer here. If it does not crash, it is only because hasSceneManager does not read from getName() due an empty scenemanager list.